faidx.c

```diff
@@ -212,8 +213,10 @@ static faidx_t *fai_read(hFILE *fp, const char *fname)
     if (!buf) goto fail;
 
     while ((l = hgetln(buf, 0x10000, fp)) > 0) {
-        for (p = buf; *p && isgraph_c(*p); ++p);
-        *p = 0; ++p;
+        for (p = buf; *p && !isspace(*p); ++p);
```

Comment:
The commentary in _hts_internal.h_ is intended to explain why it's necessary, and the commit message that added the functions notes which particular platform still emits annoying messages.  (Surprisingly enough, it's Windows.)